source/whats-new.txt

```diff
@@ -33,6 +34,30 @@ Learn what's new in:
 
 .. _upcoming-breaking-changes:
 
+.. _csharp-version-3.2:
+
+What's New in 3.2
+-----------------
+
+The 3.2 driver release includes the following new features:
+
+- Adds a new ``ObjectSerializerAllowedTypesConvention`` convention pack that
+  allows you to specify which types are allowed to be serialized and
+  deserialized by the object serializer. To learn more about convention packs,
+  see the :ref:`csharp-convention-packs` guide.
```

Comment:
Note for reviewer: This section is already reviewed/approved in a separate PR (to be merged before this one)